newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -139,7 +139,7 @@ export type SubscriptionPlanPricingSystem = {|
   period: 'week' | 'month' | 'year',
   isPerUser?: true,
   currency: 'EUR' | 'USD',
-  region: 'north' | 'south' | 'everywhere',
```

Comment:
that was... quite an interesting split 😆

Reply:
As this is returned by the backend, I think it could even be a `string`. Not sure a typing in the front is 1) useful and 2) could even be wrong as the backend could add something else.